src/main/java/com/teamknp/hotel/config/ThymeleafDialectConfig.java

```diff
@@ -0,0 +1,19 @@
+package com.teamknp.hotel.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.stereotype.Component;
+import org.thymeleaf.extras.java8time.dialect.Java8TimeDialect;
+import org.thymeleaf.extras.springsecurity4.dialect.SpringSecurityDialect;
+
+@Component
+public class ThymeleafDialectConfig {
+    @Bean
+    Java8TimeDialect java8TimeDialect() {
+        return new Java8TimeDialect();
+    }
+
+    @Bean
+    SpringSecurityDialect springSecurityDialect() {
```

Comment:
https://github.com/spring-projects/spring-boot/blob/master/spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/thymeleaf/ThymeleafAutoConfiguration.java#L321-L330
Tutaj widziałem, że jest taki bean rejestrowany, ale to nie działało mi :-/ Marcie tez ostatnio się to psuło.

Reply:
Nic mi się nie krzaczy przy korzystaniu z serwisu, więc podejrzewam, że wszystkie dialekty działają.